trogon/trogon.py

```diff
@@ -57,32 +54,28 @@ class CommandBuilder(Screen):
 
     def __init__(
         self,
-        cli: click.BaseCommand,
-        click_app_name: str,
-        command_name: str,
+        command_schemas: dict[CommandName, CommandSchema],
```

Comment:
`CommandBuilder` now accepts the schemas directly, rather than inferring schemas from a given CLI.